ngonoessono_a05a.js

```diff
@@ -0,0 +1,44 @@
+/*Gertrude Vanessa Ngono Essono
+ngonoessono_a05a.js
+INFO2124
+Thoendel
+01/172020*/
+
+const todo = [];
+todo[0]= 'cleaning';
+todo[1]= 'laundry';
+todo[2]= 'shopping';
+todo[3]= 'volunteering';
+console.log(todo);
+todo.unshift('painting');
+console.log(todo);
+
+todo.reverse();
+console.log(todo);
+
+todo.sort();
+console.log(todo);
+
+todo.includes('x');
+
+const grocery = new Array();
+grocery[0]= 'apples';
+grocery[1]= 'mangoes';
+grocery[2]= 'carrots';
+grocery[3]= 'wine';
+grocery[4]= 'chicken';
+grocery[5]= 'lemonade';
+console.log(grocery);
+
+/*there are two ways to remove the first item:
+grocery.shift() however, we are going to have 5 items in the list.
+inserting 2 others items in the middle will be difficult because the middle of 5 is 2.5.
```

Comment:
I see why you wanted to do things this way - but I would argue that a blank item is not desirable in the list. I would give you points whether the items were inserted a 2 or 3. Since you explained why you approached the problem this way - I will not dock points.